src/wallet/wallet.cpp

```diff
@@ -2097,6 +2097,9 @@ bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, bool ov
 bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wtxNew, CReserveKey& reservekey, CAmount& nFeeRet,
                                 int& nChangePosInOut, std::string& strFailReason, const CCoinControl* coinControl, bool sign)
 {
+    CFeeRate minValueRate = mempool.estimateFee(6);
```

Comment:
It's better to use the mempool relay fee here, I think. Estimatefee is
about getting into a block and depends on the speed. The relay fee is just
about being relayed.